### PR TITLE
Add Starlark parsing example of nested JSON

### PR DESCRIPTION
**Pull request by samhld**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

**Review by Jagularr — APPROVED** [review]

File: plugins/processors/starlark/testdata/json_nested.star [hunk 1]
```diff
@@ -0,0 +1,42 @@
+"""
+This code assumes the value parser with data_type='string' is used
+in the input collecting the JSON data. The entire JSON obj/doc will
+be set to a Field named `value` with which this code will work.
+
+Example input: ```
```
Comment (srebhan) [inline plugins/processors/starlark/testdata/json_nested.star:6]: Has to be 
```suggestion
Example Input: ```
```

Comment (srebhan) [reply plugins/processors/starlark/testdata/json_nested.star:6]: Furthermore I'm not sure if the current implementation of the test can cope with multi-line input. AFAIK we read one input per line, so please remove the nice formatting and make it an ugly one-liner. Never thought I will say this ever... ;-)

Comment (srebhan) [reply plugins/processors/starlark/testdata/json_nested.star:6]: Last but not least the given input/output has to be in line-protocol format... Please fix those issues to keep the starlark test-suite working...

File: plugins/processors/starlark/testdata/json_nested.star [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        "name": "partition",
+        "tags": {
+            "host": "CUD1-001559",
+            "jolokia_agent_url": "http://localhost:7777/jolokia",
+            "partition": "1",
+            "topic": "qa-kafka-connect-logs"
+        },
+        "timestamp": 1591124461
+    } ```
+Example output:
```
Comment (srebhan) [inline plugins/processors/starlark/testdata/json_nested.star:24]: Has to be `Example Output`.

**Review by srebhan — CHANGES_REQUESTED** [review]
The starlark testdata directory contains testable examples. To run those tests the `Example Input` and `Example Output` have to be in influx-line-protocol format. Please fix those. Code looks good.

**Review by srebhan — APPROVED** [review]
Thank you very much!

**Comment by samhld** [comment]
@srebhan this commit shouldn't have been part of the PR.  I still need to make the changes you mentioned to this one.  I crossed PRs accidentally.

**Comment by srebhan** [comment]
@samhld you can still change the PR and force-push it to replace it completely. I will remove the "ready-to-merge" flag for the time being. Please tell me once you are ready...

**Comment by samhld** [comment]
ok @srebhan this one should be good to go too!

File: plugins/processors/starlark/testdata/json_nested.star [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# 
+# Example Input:
+# json value="[{\"fields\": {\"LogEndOffset\": 339238, \"LogStartOffset\": 339238, \"NumLogSegments\": 1, \"Size\": 0, \"UnderReplicatedPartitions\": 0}, \"name\": \"partition\", \"tags\": {\"host\": \"CUD1-001559\", \"jolokia_agent_url\": \"http://localhost:7777/jolokia\", \"partition\": \"1\", \"topic\": \"qa-kafka-connect-logs\"}, \"timestamp\": 1591124461}]"
+
+# Example Output:
+# partition,host=CUD1-001559,jolokia_agent_url=http://localhost:7777/jolokia,partition=1,topic=qa-kafka-connect-logs LogEndOffset=339238i,LogStartOffset=339238i,NumLogSegments=1i,Size=0i,UnderReplicatedPartitions=0i 1610056029037925000
+
+
+load("json.star", "json")
+load("logging.star", "log")
```
Comment (srebhan) [inline plugins/processors/starlark/testdata/json_nested.star:34]: I guess this is not required anymore!?

**Review by srebhan — COMMENTED** [review]
Please remove the import of `logging.star`. Otherwise it's good to go.

**Comment by srebhan** [comment]
@samhld any news on this?

**Comment by samhld** [comment]
@srebhan Missed this, sorry!  Done.

**Pull request merged**